Updated screenshots

Ieno77 opened the pull request:
# Description
Updated screenshots 

# Impacts
Documentations


# Delete branch after merge
(REQUIRED)
YES

# Issue
fixes #5054
fixes #5055

nqb commented on docs/PacketFence_Installation_Guide.asciidoc:
```diff
@@ -3758,11 +3758,7 @@ Notes on the configuration:
-image::rapid7/rapid7-pf-2.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
-
-image::rapid7/rapid7-pf-3.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
+image::rapid7/rapid7-pf.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
```
Remove old rapid7-pf-X files if they are not used anymore.

Ieno77 replied:
done

nqb commented on docs/PacketFence_Installation_Guide.asciidoc:
```diff
@@ -4123,9 +4119,7 @@ Next, go in the 'NTLM cache' tab and:
-image::domain-ntlm-cache-1.png[scaledwidth="100%",alt="Domain NTLM cache"]
-
-image::domain-ntlm-cache-2.png[scaledwidth="100%",alt="Domain NTLM cache"]
+image::domain-ntlm-cache.png[scaledwidth="100%",alt="Domain NTLM cache"]
```
- Remove old domain-ntlm-cache-X files if they are not used anymore.
- GitHub report a conflict on domain-ntlm-cache.png that prevent a merge

nqb commented on docs/pki/microsoft.asciidoc:
```diff
@@ -229,15 +229,13 @@ Add a new PKI provider and select SCEP.
 
 You do not need any Username/Password combination for this configuration.
 
-image::scep-ms-pki-provider2.png[scaledwidth="100%",alt="MSPKI configuration"]
```
Remove this file if not needed anymore.

nqb commented on docs/pki/packetfence.asciidoc:
```diff
@@ -321,8 +321,6 @@ Pay attention to the username and password which have to match an authorized use
 
 image::packetfence-pki-provider.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
 
-image::packetfence-pki-provider2.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
```
Remove this file if not needed anymore.

nqb commented:
Hello @Ieno77, 

Could you update description of this PR with links to issues related to images (like #5055 and #5054) ?

Thanks

nqb commented:
@Ieno77, you certainly need to rebase this PR.

julsemaan commented:
@nqb, can you review again

cgx commented:
Don't forget to [crunch](https://github.com/chrissimpkins/Crunch) your images.

nqb commented:
Need to be rebased after merge of #5170. 

leno77, could you crunch your images ?

cgx commented:
@Ieno77 Good! Now it's time to squash your commits.

nqb commented:
@Ieno77, could you rebase ?

Ieno77 commented:
Please @nqb can you review and merge ?

cgx commented:
@Ieno77 Please use lowercase letters for all filenames.

Ieno77 commented:
@nqb I removed useless screens and changed the aruba-registration-acl
For the dpsk_provisionner, the old screen was already showing a Windows provisionner

nqb commented:
>  For the dpsk_provisionner, the old screen was already showing a Windows provisionner

It makes sense regarding location of this screen in documentation.
